src/providers/fault-count/fault-count.ts

```diff
@@ -80,6 +80,7 @@ export class FaultCountProvider {
       case TestCategory.C: return FaultCountCHelper.getVehicleChecksFaultCountCatC(data);
       // TODO: To be implemented properly in MES-4420
       case TestCategory.EUAM1: return FaultCountBEHelper.getVehicleChecksFaultCountCatBE(data);
+      case TestCategory.EUAM2: return FaultCountBEHelper.getVehicleChecksFaultCountCatBE(data);
```

Comment:
no EUAM1 and EUAM2 specific function?

Reply:
MES-4420 should take care of correctly implementing the functionality for Cat A